app/translations/en.json

```
@@ -991,8 +991,8 @@
             "noDataMessage": "There are no Blueprints available in catalog. Check widget's configuration.",
             "catalog": {
                 "properties": {
-                    "created": "Created",
-                    "updated": "Updated"
+                    "created": "Created:",
+                    "updated": "Updated: "
```

Minor thing - I suspect we don't need space here:
```suggestion
                    "updated": "Updated:"
```

---

fixed: https://github.com/cloudify-cosmo/cloudify-stage/pull/2197/commits/78e3f858954df8f13e5240da997fab051143ca12